package-lock.json

```diff
@@ -0,0 +1,2129 @@
+{
```

Comment:
I'd run `npm install` in the root to make sure this is consistent with the changes in `package.json`